sdk/storage/azblob/appendblob/client.go

```diff
@@ -36,11 +36,13 @@ func NewClient(blobURL string, cred azcore.TokenCredential, options *ClientOptio
 	authPolicy := shared.NewStorageChallengePolicy(cred)
 	conOptions := shared.GetClientOptions(options)
 	conOptions.PerRetryPolicies = append(conOptions.PerRetryPolicies, authPolicy)
```

Comment:
The `authPolicy` should go in the `runtime.PipelineOptions.PerRetryPolicies` field (this was missed in earlier reviews).

Reply:
Fixed!